minimum_versions/environments/pixi.py

```diff
@@ -0,0 +1,128 @@
+import pathlib
+import re
+
+import tomllib
+from rattler import Version
+from tlz.dicttoolz import get_in, merge
+
+from minimum_versions.environments.spec import Spec
+
+_version_re = r"[0-9]+\.[0-9]+(?:\.[0-9]+|\.\*)?"
+version_re = re.compile(f"(?P<version>{_version_re})")
+lower_pin_re = re.compile(rf">=(?P<version>{_version_re})$")
+tight_pin_re = re.compile(rf">=(?P<lower>{_version_re}),<(?P<upper>{_version_re})")
+
+
+def parse_spec(name, version_text):
+    # "*" => None
```

Comment:
I've been testing with the xarray manifest, and all the minimum envs there have the 'test' feature, which has 
```
array-api-strict = "<2.4"
```
as a dep, but this errors with 


```
ValueError: Unknown version format: <2.4
```
since this function doesn't support "<" as a specifier

Reply:
Actually, I realised that the env I was looking at was the wrong one (`test-py311-bare-min-and-scipy` when it should be `test-py311-min-versions` - the latter which has `array-api-strict` pinned to something else). So I don't think this is a problem

Reply:
it actually is a problem (but not with this action, see below): `array-api-strict` comes from the `test` feature and is included in `test-py311-bare-minimum`, which we definitely want to check.

However, I would argue that `array-api-strict` is not part of the test deps (like `pytest` or `hypothesis`), but rather a dependency that should be part of the `extras` feature. Which means the pin would not be included in the min versions envs, side-stepping this issue.